sdgym/benchmark.py

```diff
@@ -151,75 +129,52 @@ def _compute_scores(metrics, real_data, synthetic_data, metadata, output, comput
 
     if compute_quality_score:
         start = datetime.utcnow()
-        if metadata.modality == 'single-table':
```

Comment:
are we getting rid of the modality checks because this PR will drop support for other modalities?

Reply:
In this PR I tried to make the minimal changes to make the code work. I didn't want to completely delete the support for other modalities here. I think if we want to do that it should be done in a different PR.

Another option is to not merge this PR to master, and instead to some other branch `ready-for-release`, which in turn will be merged to master. We can then make all the other incremental changes there, if you think merging code with pieces that don't work is a problem.